lib/cli/commands/openchannel.ts

```diff
@@ -0,0 +1,31 @@
+import { Arguments } from 'yargs';
+import { callback, loadXudClient } from '../command';
+import { OpenChannelRequest } from '../../proto/xudrpc_pb';
+import { coinsToSats } from '../utils';
+
+export const command = 'openchannel <nodePubKey> <currency> <amount>';
+
+export const describe = 'open a payment channel';
+
+export const builder = {
+  nodePubKey: {
+    description: 'The peer public key which to open channel with',
```

Comment:
How about "The node pub key of the peer to open the channel with"?

Reply:
Yep, that sounds better :).